src/onepasswordconnectsdk/client.py

```diff
@@ -368,9 +368,9 @@ def get_vaults(self):
                      for {url} with message {response.json().get('message')}"
             )
 
-        return self.deserialize(response.content, "list[Vault]")
+        return self.deserialize(response.content, list[Vault])
```

Comment:
Unfortunately, `deserialize` won't work if the provide type is as it is and not as a string.

Reply:
I don't think this change was intentional.

I'd love to see this PR merged (so I don't have to create a similar PR), could a maintainer please revert this one line and merge the PR?

Reply:
Sorry, I missed this feedback! I'll fix it 😉

Reply:
Reverted all deserialize calls to strings!